packages/cli/src/requests.ts

```diff
@@ -601,3 +601,10 @@ export declare namespace ProjectRequest {
 	>;
 	type Delete = AuthenticatedRequest<{ projectId: string }, {}, {}, { transferId?: string }>;
 }
+
+// ----------------------------------
+//           /nps-survey
+// ----------------------------------
+export declare namespace NpsSurveyRequest {
+	type NpsSurveyUpdate = AuthenticatedRequest<{}, {}, unknown>;
```

Comment:
For documentation it would be best to have the valid body type defined instead of using `unknown`. If/When we have a schema for the request there are libraries that give the type automatically based on that. (zod & typebox at least). For now we would have to write it manually

Reply:
Okay can add a comment